pycaption/base.py

```diff
@@ -225,18 +257,15 @@ class CaptionSet(object):
     The .layout_info attribute, keeps information that should be inherited
     by all the children.
     """
-    def __init__(self):
-        self._styles = {}
-
-        # Base layout to be inherited by all languages
-        self.layout_info = None
-
-        # For individual languages, represents inheritable layout-related
-        # information
-        self._layout_info = {}
-
-        # Captions by language.
-        self._captions = defaultdict(list)
+    def __init__(self, captions, styles={}, layout_info=None):
+        """
+        :param captions: A dictionary of the format {'language': [CaptionList]}
```

Comment:
Shouldn't this be {'language': CaptionList} ? (the value is not a list containing a CaptiopnList, but simply a CaptionList)